Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -155,18 +155,18 @@ public class SessionController : BaseJellyfinApiController
         /// <param name="startPositionTicks">The starting position of the first item.</param>
         /// <response code="204">Instruction sent to session.</response>
         /// <returns>A <see cref="NoContentResult"/>.</returns>
-        [HttpPost("Sessions/{sessionId}/Playing/{command}")]
+        [HttpPost("Sessions/{sessionId}/Playing")]
         [Authorize(Policy = Policies.DefaultAuthorization)]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         public ActionResult Play(
             [FromRoute, Required] string sessionId,
             [FromRoute, Required] PlayCommand command,
-            [FromQuery] Guid[] itemIds,
+            [FromQuery] Guid itemIds,
```

Comment:
itemIds is a comma-delimited string................

```suggestion
            [FromQuery] string itemIds,
```